search/src/test/scala/weco/api/search/models/QueryConfigTest.scala

```diff
@@ -63,14 +63,5 @@ class QueryConfigTest
         result.paletteBinSizes shouldBe QueryConfig.defaultPaletteBinSizes
       }
     }
-
-    it("returns the default config if the data is not found") {
```

Comment:
Is this no longer desirable?  Should this be replaced with a test for the desired behaviour in this scenario, or is it now impossible to reach?

Reply:
Impossible to reach – this is from when inferredData was an optional field on images and so might be `None`, but in practice we know it's always going to be populated. See https://github.com/wellcomecollection/catalogue-pipeline/pull/2219